### PR TITLE
Fixes blocking changes for mkdocs site migration

### PR DESCRIPTION
**Pull request by ta11t0m**
- Add an Hardware index overview
- Reformat and add images to Hardware getting started
- Rename 1CC Getting Started to index
- Add missing SSH image needed for 1CC getting started 
- Add a page for 1CC Support

### CODE REVIEW

File: docs/public-cloud/1-click-clusters/support.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Support team
+
+When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:
+
+* **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.
+* **Technical Account Manager(TAM)**: An expert specialist who understands the intricacies of your deployment, provides ongoing technical guidance, and escalates any complex issues. 
+* **Machine learning expert (MLE)**: A AI/ML expert within lambda will provide guidance how how to integrate and scale AI workloads.    
+* **Support Engineering**: Lambda’s Support team, available 24/7 via our ticketing portal, is well-versed in 1CC and will be able to respond to any technical support request, escalating issues and incidents early and often.
+
+## Support Scope
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:21]:
```suggestion
## Support scope
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Lambda classifies incoming support tickets into three categories:
+
+* In scope
+* Best effort
+* Out of scope
+
+### In Scope
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:29]:
```suggestion
### In scope
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+* **Hardware and Infrastructure:** Full support for CPU/GPU VMs, physical hosts, and networking components.  
+* **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and OpenMPI.  
+* **Networking and Storage:** Management and troubleshooting for Ethernet and InfiniBand networks, as well as persistent and local storage.  
+* **Slurm Installation:** Guidance on Slurm setup and basic troubleshooting to streamline your job scheduling processes.  
+* **Managed Kubernetes*:** If purchased as an add-on, our team of Kubernetes experts will be there to help you.
+
+### Best Effort Support:
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:37]:
```suggestion
### Best effort support:
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+At Lambda, we recognize that exceptional support is critical to maximizing the value of your One-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
+
+## Support team
+
+When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:
+
+* **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.
+* **Technical Account Manager(TAM)**: An expert specialist who understands the intricacies of your deployment, provides ongoing technical guidance, and escalates any complex issues. 
+* **Machine learning expert (MLE)**: A AI/ML expert within lambda will provide guidance how how to integrate and scale AI workloads.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:18]:
```suggestion
* **Machine Learning Expert (MLE)**: An AI/ML expert within Lambda will provide guidance on how to integrate and scale AI workloads.    
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+# Support
+
+At Lambda, we recognize that exceptional support is critical to maximizing the value of your One-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
+
+## Support team
+
+When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:
+
+* **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.
+* **Technical Account Manager(TAM)**: An expert specialist who understands the intricacies of your deployment, provides ongoing technical guidance, and escalates any complex issues.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:17]:
```suggestion
* **Technical Account Manager(TAM)**: An expert specialist who understands the specifics of your deployment, provides ongoing technical guidance, and escalates any complex issues. 
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+# Support
+
+At Lambda, we recognize that exceptional support is critical to maximizing the value of your One-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
+
+## Support team
+
+When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:
+
+* **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:16]:
```suggestion
* **Customer Success Manager (CSM)**: Your main point of contact post-sales, responsible for ensuring the delivery of your solution and your overall satisfaction.
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+description: How to access 1-Click Clusters
+tags:
+  - 1-click-clusters
+  - support
+---
+
+# Support
+
+At Lambda, we recognize that exceptional support is critical to maximizing the value of your One-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:10]:
```suggestion
At Lambda, we recognize that exceptional support is critical to maximizing the value of your 1-Click Cluster (1CC) deployment. Our world-class support team is dedicated to ensuring your success at every stage, from deployment to daily operations.
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+## Support team
+
+When you choose Lambda, you gain access to a dedicated support team of seasoned professionals with deep expertise in AI/ML infrastructure. This team includes:
+
+* **Customer Success Manager (CSM)**: Your main point of contact post sales, responsible for ensuring your overall satisfaction and delivery of your solution.
+* **Technical Account Manager(TAM)**: An expert specialist who understands the intricacies of your deployment, provides ongoing technical guidance, and escalates any complex issues. 
+* **Machine learning expert (MLE)**: A AI/ML expert within lambda will provide guidance how how to integrate and scale AI workloads.    
+* **Support Engineering**: Lambda’s Support team, available 24/7 via our ticketing portal, is well-versed in 1CC and will be able to respond to any technical support request, escalating issues and incidents early and often.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:19]:
```suggestion
* **Support Engineering**: Lambda’s Support team, available 24/7 through our ticketing portal, is well-versed in 1CC and will be able to respond to any technical support request, escalating issues and incidents early and often.
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+Lambda classifies incoming support tickets into three categories:
+
+* In scope
+* Best effort
+* Out of scope
+
+### In Scope
+
+* **Hardware and Infrastructure:** Full support for CPU/GPU VMs, physical hosts, and networking components.  
+* **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and OpenMPI.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:32]:
```suggestion
* **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and Open MPI.  
```

File: docs/public-cloud/1-click-clusters/support.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+* Best effort
+* Out of scope
+
+### In Scope
+
+* **Hardware and Infrastructure:** Full support for CPU/GPU VMs, physical hosts, and networking components.  
+* **Software Environment:** Assistance with Lambda Stack, OFED drivers, Jupyter Notebooks, and essential ML tools like NCCL and OpenMPI.  
+* **Networking and Storage:** Management and troubleshooting for Ethernet and InfiniBand networks, as well as persistent and local storage.  
+* **Slurm Installation:** Guidance on Slurm setup and basic troubleshooting to streamline your job scheduling processes.  
+* **Managed Kubernetes*:** If purchased as an add-on, our team of Kubernetes experts will be there to help you.
```
Comment (ryanlambda) [inline docs/public-cloud/1-click-clusters/support.md:35]: Are we sure we want to word it this way? It seems to promise a level of support that we don't necessarily want to be promising.

Comment (ta11t0m) [reply docs/public-cloud/1-click-clusters/support.md:35]: This language has been blessed by Matt Hammons.

**Review by ryanlambda — CHANGES_REQUESTED** [review]
Added some clean-up suggestions to the Support doc. Otherwise, LGTM.

**Review by cbrownstein-lambda — APPROVED** [review]

**Review by ryanlambda — APPROVED** [review]

**Pull request merged**